Composer/packages/client/src/pages/language-generation/index.tsx

```diff
@@ -215,7 +228,7 @@ const LGPage: React.FC<RouteComponentProps> = props => {
           <div css={contentEditor}>
             {editMode ? (
               <Suspense fallback={<LoadingSpinner />}>
-                <CodeEditor file={lgFile} codeRange={codeRange} onChange={onChange} />
+                <CodeEditor file={lgFile} codeRange={codeRange} onChange={onChange} onMount={setLgEditor} />
```

Comment:
how about  do this in code-editor.tsx? it read url hash,  let index.tsx  aware `#line1`  logic from editor may not be necessary ?